New_PlatformGame/Assets/UIScript.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Net.Mime;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UIScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public Text scoreText;
+    private int _score;
+    public GameObject player;
```

Comment:
```suggestion
    public Text scoreText;
    private int _score;
    public GameObject player;
```

public은 public끼리 private은 private끼리